node/src/reactor/initializer.rs

```diff
@@ -59,11 +60,15 @@ pub(crate) enum Event {
     /// Storage request.
     #[from]
     StorageRequest(StorageRequest),
+
+    /// Contract runtime request.
+    #[from]
+    ContractRuntimeRequest(Box<ContractRuntimeRequest>),
 }
 
 impl From<ContractRuntimeRequest> for Event {
-    fn from(contract_runtime_request: ContractRuntimeRequest) -> Self {
-        Event::ContractRuntime(Box::new(contract_runtime_request))
+    fn from(request: ContractRuntimeRequest) -> Self {
+        Event::ContractRuntimeRequest(Box::new(request))
```

Comment:
As stated above, I am not a fan of these boxing, as I think they're redundant/better done on the type itself.

Reply:
It's gone with the https://github.com/casper-network/casper-node/commit/161222fffeefa683ccbfd5e57e554d59cefdb734